repo/plugin.video.nzbdav/resources/lib/resolver.py

```
@@ -3739,66 +3754,95 @@ def _advertised_size_bytes(download_size):
+    # disambiguates a job-start stub from a legitimate pack episode.
+    if picked_size >= floor:
+        return False
+    # Picked file is small (or its size is unknown): could be the job-start stub
+    # OR a legitimate pack episode. Disambiguate by the folder's total real video
+    # content -- a real pack's episodes sum to ~advertised and pass; a stub-only
+    # folder falls far below and is rejected.
+    try:
+        total = _webdav.folder_video_total_bytes(
+            webdav_folder, settings_getter=settings_getter
+        )
     except Exception:  # pylint: disable=broad-except
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)